geoextract/app.py

```diff
@@ -57,9 +57,9 @@ def extract():
         try:
             text = request.files['text'].read().decode('utf-8')
         except KeyError:
-            abort(400, 'Missing "text" parameter.')
+            return abort(400, 'Missing "text" parameter.')
```

Comment:
According to the [documentation](http://flask.pocoo.org/docs/0.12/api/#flask.abort), `abort` raises an exception, so there's no point in adding a `return`.